File: examples/c/document_save_to_path.c
#include <gxml/gxml.h>
#include <stdio.h>

int main () {
  GXmlDocument *doc;

  doc = gxml_document_new_from_path ("bookshelf.xml");

  gxml_document_save_to_path (doc, "output/bookshelf_save_to_path.xml");

  g_object_unref (doc);

  return 0;
}

File: examples/js/document_save_to_path.js
#!/usr/bin/gjs

const GXml = imports.gi.GXml;
const Gio = imports.gi.Gio;

function saving_a_document_to_a_path () {
    let doc = GXml.GomDocument.from_path ("bookshelf.xml");
    var f = Gio.File.new_for_path ("bookshelf_save_to_path.xml")
    doc.write_file (f);
    if (!f.query_exists (null)) print ("Can't create file");
}

saving_a_document_to_a_path ();
